swagger.json

```diff
@@ -2574,9 +2574,234 @@
+                        "in": "path",
+                        "required": true,
+                        "description": "ID of the address/geofence",
+                        "type": "integer",
+                        "format": "int64"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Address was successfully deleted. No response body is returned."
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
         }
     },
     "definitions": {
+        "Address": {
+            "type": "object",
+            "description": "Information about an address/geofence. Geofences are either a circle or a polygon.",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "ID of the address",
+                    "format": "int64",
+                    "example": 123
+                },
+                "name": {
+                    "type": "string",
+                    "description": "Name of the address or geofence",
+                    "example": "Samsara HQ"
+                },
+                "formattedAddress": {
+                    "type": "string",
+                    "description": "The full address associated with this address/geofence, as it might be recognized by maps.google.com",
+                    "example": "350 Rhode Island St, San Francisco, CA"
+                },
+                "geofence": {
+                    "$ref": "#/definitions/AddressGeofence"
+                }
+            }
+        },
+        "AddressGeofence": {
+            "type": "object",
+            "description": "The geofence that defines this address and its bounds. This can either be a circle, or a polygon - only one key should be provided, depending on the geofence type.",
+            "properties": {
+                "circle": {
+                    "type": "object",
+                    "description": "Information about a circular geofence. This field is only populated if the geofence is a circle.",
+                    "properties": {
+                        "latitude": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "The latitude of the center of the circular geofence",
+                            "example": "37.765363"
+                        },
+                        "longitude": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "The longitude of the center of the circular geofence",
+                            "example": "-122.403098"
```

Comment:
@silverspace It looks like the example gets rendered as-is (without checking the type). Should the type for this example be a float instead of a string?

Reply:
That's a good catch, this should be a float value.

Reply:
I'll make this update!